honeybee_vtk/load_config.py

```diff
@@ -0,0 +1,236 @@
+        grids_info = json.load(fh)
+
+    # TODO: Make sure to remove this limitation. A user should not have to always
+    # TODO: load all the grids
+    assert len(model.sensor_grids.data) == len(grids_info), 'The number of result'\
+        f' files {len(grids_info)} does for {data.identifier} does not match'\
+        f' the number of sensor grids in the model {len(model.sensor_grids.data)}.'
+
+    # match identifiers of the grids with the identifiers of the result files
+    grids_model_identifiers = [grid.identifier for grid in model.sensor_grids.data]
+    grids_info_identifiers = [grid['full_id'] for grid in grids_info]
+    assert grids_model_identifiers == grids_info_identifiers, 'The identifiers of'\
+        ' the sensor grids in the model do not match the identifiers of the grids'\
+        f' in the grids_info.json for {data.identifier}.'
+
+    # make sure length of each file matches the number of sensors in grid
+    file_lengths = [grid['count'] for grid in grids_info]
+
+    # check if the grid data is meshes or points
+    # if grid is sensors
+    if grid_type == 'points':
+        num_sensors = [polydata.GetNumberOfPoints()
+                       for polydata in model.sensor_grids.data]
+    # if grid is meshes
+    else:
+        num_sensors = [polydata.GetNumberOfCells()
+                       for polydata in model.sensor_grids.data]
+
+    # lastly check if the length of a file matches the number of sensors or meshes on grid
+    if file_lengths != num_sensors:
+        length_matching = {
+            grids_info_identifiers[i]: file_lengths[i] == num_sensors[i] for i in
+            range(len(grids_model_identifiers))
+        }
+        names_to_report = [
+            id for id in length_matching if length_matching[id] is False]
+        raise ValueError(
+            'File lengths of result files must match the number of sensors on grids.'
+            ' Lengths of files with following names do not match'
+            f' {tuple(names_to_report)}.')
+
+
+def _load_legend_parameters(data: DataConfig, model: Model, scene: Scene) -> None:
+    """Load legend_parameters.
+
+    Args:
+        data: A Dataconfig object.
+        model: A honeybee-vtk model object.
+        scene: A honeyebee-vtk scene object.
+    """
+    legend_params = data.legend_parameters
+    legend = scene.legend_parameter(data.identifier)
+
+    legend.colors = legend_params.color_set
+    legend.unit = data.unit
+
+    if isinstance(legend_params.min, Autocalculate):
+        legend.min = None
+    else:
+        legend.min = legend_params.min
+
+    if isinstance(legend_params.max, Autocalculate):
+        legend.max = None
+    else:
+        legend.max = legend_params.max
+
+    if not legend.min and not legend.max:
+        warnings.warn(
+            f'In data {data.identifier.capitalize()}, since min and max'
+            ' values are not provided, those values will be auto calculated based'
+            ' on data. \n'
```

Comment:
It feels a bit wordy what about this:

```suggestion
            f'Legend min and max values are autocalculated for {data.identifier.capitalize()}.\n'
```

Also I don't know if we want to give this warning at all. It might confuse our users more than being helpful.